docs/source/get-started/configuration-guide.rst

```diff
@@ -773,6 +773,10 @@ Intel GPUs
       * Iris Xe MAX (DG1)
       *
 
+    * * ``Kokkos_ARCH_INTEL_DG2``
```

Comment:
Do you want to mention since what Kokkos version it is available?

Reply:
Sure. I added a column.